lib/queueservice.js

```diff
@@ -107,12 +107,10 @@ export default class TaskQueue {
     let claim;
     try {
       claim = await this.stats.timeGen(
-        'tasks.time.claim',
+        'claimTask',
```

Comment:
so far, the stats added to the client seem to be exactly what we're going after, still reviewing, but I think once the new client lands we could change these a bit